stable/aws-tigera-operator/README.md

```diff
@@ -0,0 +1,37 @@
+# Calico on AWS
+
+This chart installs Calico Operator on AWS: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
+
+## Prerequisites
+
+- Kubernetes 1.11+ running on AWS
+
+## Installing the Chart
+
+First add the EKS repository to Helm:
+
+```shell script
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+Install the Tigera Operator charts with the release name `aws-tigera-operator` and default configuration:
+```shell script
+helm install aws-tigera-operator eks/aws-tigera-operator
+```
+To install into an EKS cluster where the CNI is already installed, you can run:
+```shell script
+helm upgrade --install --recreate-pods --force aws-tigera-operator eks/aws-tigera-operator
+```
+Note:
+- The Tigera operator installs Calico CRDs.
+- The Tigera operator installs resources in the calico-system namespace. Previous aws-calico install methods use the kube-system namespace instead.
```

Comment:
You may want to mention that the operator install will migrate a calico previously installed in kube-system to calico-system and remove the old daemonset when finished.

I'm not sure what the process would be to upgrade from the previous `aws-calico` helm chart to this one because I don't know the internals of helm and what will happen since the `aws-calico` chart had installed the CRDs and the `aws-tigera-operator` would be taking them over. It might be an easier path if they were both named `aws-calico`.